test/test.js

```diff
@@ -22,6 +22,14 @@ var MOCK_JS_VARIABLES =  {
 	'js-defined-no-prefix': '#ff0000'
 };
 
+var INVALID_JS_VARIABLES = {
+	'number-value': 50,
+	'zero-value': 0,
+	'null-value': null,
+	'undefined-value': undefined,
+	'object-value-passed-by-mistake': {}
+};
```

Comment:
We should add a `Boolean`(`true`/`false`)

Reply:
Agree, done in shonie@2c651e8f3df1b835aa023375d74a2306c9bfef96